frontend/src/screens/BackupMnemonic.tsx

```diff
@@ -149,34 +157,64 @@ export function BackupMnemonic() {
                 <ShieldCheck className="w-6 h-6" />
               </div>
               <span>
-                Make sure to write them down somewhere safe and private
+                Make sure to write them down somewhere safe and private.
               </span>
             </div>
             <div className="flex gap-2 items-center text-destructive">
               <div className="shrink-0 ">
                 <ShieldAlert className="w-6 h-6" />
               </div>
               <span>
-                If you lose your recovery phrase, you will lose access to your
-                funds
+                If you lose access to your hub and do not have your{" "}
+                <b>recovery phrase</b>&nbsp;
+                {info?.albyAccountConnected && (
+                  <>
+                    or your <b>unlock password</b>
+                  </>
+                )}
+                {!info?.albyAccountConnected && (
+                  <>or do not backup your data directory</>
+                )}
+                , you will lose access to your funds.
               </span>
             </div>
           </div>
+          <div className="mb-5">
+            <ExternalLink
+              className="underline flex items-center"
+              to="https://guides.getalby.com/user-guide/v/alby-account-and-browser-extension/alby-hub/backups"
+            >
+              Learn more about backups
+              <ExternalLinkIcon className="w-4 h-4 ml-2" />
+            </ExternalLink>
+          </div>
 
           <MnemonicInputs mnemonic={decryptedMnemonic} readOnly={true}>
             <div className="flex items-center mt-5">
               <Checkbox
                 id="backup"
+                required
                 onCheckedChange={() => setIsBackedUp(!backedUp)}
               />
               <Label htmlFor="backup" className="ml-2">
                 I've backed up my recovery phrase to my wallet in a private and
                 secure place
               </Label>
             </div>
+            <div className="flex items-center mt-5">
+              <Checkbox
+                id="backup2"
+                required
+                onCheckedChange={() => setIsBackedUp2(!backedUp2)}
+              />
+              <Label htmlFor="backup2" className="ml-2">
+                I understand the recovery phrase alone will not restore funds
+                from my lightning channels.{" "}
```

Comment:
can we say this active with the necessary details? 

I understand the unlock password AND the recovery phrase is needed to recover funds
from my lightning channels.{" "}

something like this? 

otherwise you read it and at that point don't get the necessary info.

Reply:
People tend to remember the words they read, and positive affirmations resonate better than negative ones, aligning perfectly with Bumi's suggestion.